xla/stream_executor/sycl/BUILD

```diff
@@ -85,3 +86,28 @@ cc_library(
         "//xla/stream_executor:scratch_allocator",
     ] + if_static([":all_runtime"]),
 )
+
+cc_library(
+    name = "sycl_solver_context",
+    srcs = ["sycl_solver_context.cc"],
+    hdrs = ["sycl_solver_context.h"],
+    tags = [
+        "gpu",
+        "manual",
+        "sycl-only",
```

Comment:
The CI wants all tags to be documented. Here is the error message:

```
ValueError: Tag(s) {'no-sycl', 'sycl-only'} are undocumented! Please document them in `build_tools/lint/tags.py`.
```

Can you take care of that?